shopware/core/6.4/manifest.json

```diff
@@ -51,36 +51,13 @@
         ".env.local",
         ".env.local.php",
         ".env.*.local",
-        "custom/*",
-        "!custom/plugins/.gitkeep",
-        "!custom/static-plugins/",
-        "/public/fonts/*",
-        "!/public/fonts/.gitkeep",
-        "/public/img/*",
-        "!/public/img/.gitkeep",
-        "/public/css/*",
-        "!/public/css/.gitkeep",
-        "/public/js/*",
-        "!/public/js/.gitkeep",
-        "/public/media/*",
-        "!/public/media/.gitkeep",
-        "/public/theme/*",
-        "!/public/theme/.gitkeep",
-        "/public/thumbnail/*",
-        "!/public/thumbnail/.gitkeep",
-        "/public/assets/*",
-        "/public/sitemap/*",
-        "!/public/sitemap/.gitkeep",
-        "/public/bundles/*",
-        "!/public/bundles/.gitkeep",
         "/files/*",
+        "!/files/.htaccess",
         "/var/*",
-        "!/var/log/.gitkeep",
-        "!/var/cache/.gitkeep",
+        "!/var/.htaccess",
         "/auth.json",
         "/install.lock",
-        "public/asset-manifest.json",
-        "files/asset-manifest.json"
```

Comment:
I think this should stay, as some older Shopware versions sometimes write into files. (only 6.4 version)

Reply:
Understood, I added back files/asset-manifest.json